.github/workflows/go_lint_and_test.yml

```diff
@@ -0,0 +1,27 @@
+name: golangci-lint
+on: [push, pull_request]
+#  push:
+#    branches: [ main ]
+#    paths: '**.go'
```

Comment:
Should be `**/**.go`

Reply:
https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#onpushpull_requestpaths agrees with the current code.

Reply:
Ok